src/develop/pixelpipe_hb.c

```diff
@@ -960,7 +960,7 @@ static void _pixelpipe_pick_primary_colorpicker(dt_develop_t *dev, const float *
 
   // display rgb --> histogram rgb
   if(display_profile && histogram_profile)
-    xform_rgb2rgb = cmsCreateTransform(display_profile, TYPE_RGB_FLT, histogram_profile, TYPE_RGB_FLT, INTENT_PERCEPTUAL, 0);
+    xform_rgb2rgb = cmsCreateTransform(display_profile, TYPE_RGB_FLT, histogram_profile, TYPE_RGB_FLT, INTENT_RELATIVE_COLORIMETRIC, 0);
```

Comment:
Likewise, `DT_INTENT_RELATIVE_COLORIMETRIC`

Reply:
Likewise, since a call into LCMS, should use LCMS constant, no change needed.

Reply:
No change needed here either. :)